epick_driver/include/epick_driver/default_driver.hpp

```diff
@@ -55,9 +55,11 @@ class DefaultDriver : public Driver
 
   void set_slave_address(const uint8_t slave_address) override;
   void set_mode(const GripperMode gripper_mode) override;
-  void set_max_vacuum_pressure(const float vacuum_pressure) override;
-  void set_min_vacuum_pressure(const float vacuum_pressure) override;
-  void set_gripper_timeout(const std::chrono::milliseconds gripper_timeout) override;
+  void set_grip_max_vacuum_pressure(const float vacuum_pressure) override;
```

Comment:
Does the const-ness of the argument really do anything in the declaration? If so, maybe we could remove it from here since it's just visual noise, but leave it in the implementation to prevent mutation of the parameter inside the function.

Reply:
Makes sense. Removed.